data/src/main/java/com/microsoft/azure/kusto/data/ConnectionStringBuilder.java

```diff
@@ -63,4 +63,8 @@ public static ConnectionStringBuilder createWithAadApplicationCredentials(String
     {
         return createWithAadApplicationCredentials(resourceUri, applicationClientId, applicationKey, null);
     }
+
+    public static ConnectionStringBuilder createWithDeviceCredentials(String resourceUri){
```

Comment:
I think we should keep using the same term across. How does ADAL call this - DeviceCode? if yes, I think we should us this term everywhere...

Reply:
Changed it